README.md

```diff
@@ -1,4 +1,53 @@
-Otter Browser
-=============
+# ![Otter Browser logo](resources/icons/otter-browser-64.png) Otter Browser
+
+
+***A browser controlled by the user, not vice-versa***
+
+[**Otter Browser**](https://otter-browser.org/) aims to recreate the best aspects of **Opera 12** and to revive its spirit. We are working on providing the powerful features power users want while keeping the browser fast and lightweight. We also learned from History and decided to release the browser under the GNU GPL v3.
+
+[![SourceForge](https://img.shields.io/sourceforge/dt/otter-browser.svg)](https://sourceforge.net/projects/otter-browser/files/)
+
+[![Screenshot](https://otter-browser.org/screenshots/1.png)](https://otter-browser.org/screenshots/)
+
+Planned features are listed in the *TODO* file. Details on already implemented features are available in the *CHANGELOG* file. If you have an idea that has not yet been proposed or refused, feel free to [open a new issue](https://github.com/OtterBrowser/otter-browser/issues/new).
+
+## Install
+
+You can either compile Otter Browser from source or use pre-compiled binaries.
+
+### From source
+
+To build Otter, you will need the following depedencies: Qt (at least 5.4.0), Sonnet, and CMake (3.1.0 or newer). At the root of the directory were the source code is stored, execute these commands:
```

Comment:
Sonnet is not a dependency, we use bundled version, modified for our needs.
Also I would change "To build Otter" to "To build Otter Browser" to make it more consistent.